test/arcade/potts/util/PlaneTest.java

```diff
@@ -0,0 +1,107 @@
+package arcade.potts.util;
+
+import org.junit.Test;
+import sim.util.Int3D;
+import arcade.potts.env.location.Voxel;
+import static org.junit.Assert.*;
+
+public class PlaneTest {
+
+    @Test
+    public void constructor_givenPointAndVector_returnsCorrectPlane() {
+        Voxel point = new Voxel(1, 2, 3);
+        Int3D normalVector = new Int3D(4, 5, 6);
+
+        Plane plane = new Plane(point, normalVector);
+
+        assertEquals(point, plane.referencePoint);
+        assertEquals(normalVector, plane.normalVector);
+    }
+
+    @Test
+    public void equals_givenDifferentPlane_returnsFalse() {
+        Plane plane1 = new Plane(new Voxel(1, 2, 3), new Int3D(4, 5, 6));
+        Plane plane2 = new Plane(new Voxel(4, 5, 6), new Int3D(7, 8, 9));
+
+        assertFalse(plane1.equals(plane2));
```

Comment:
```suggestion
        assertNotEquals(plane1, plane2);
```

Where possible, avoid additional calls inside the assertion